lib/contentful.js

```diff
@@ -74,53 +79,57 @@ module.exports.getClient = function getClient() {
  */
 module.exports.contentfulError = function contentfulError(error) {
   const scope = error;
-  scope.message = `Contentful: ${error.message}`;
+  scope.message = `${ERROR_PREFIX} ${error.message}`;
 
   return scope;
 };
 
+module.exports.getFirstItemFromArray = function getFirstItemFromArray(array) {
```

Comment:
If we're abstracting this -- does it make sense to just add https://lodash.com/docs/4.17.4#head?